doc/api/packages.md

```diff
@@ -1068,5 +1109,6 @@ This field defines [subpath imports][] for the current package.
 [subpath exports]: #packages_subpath_exports
 [subpath imports]: #packages_subpath_imports
 [subpath patterns]: #packages_subpath_patterns
+[supported package managers]: corepack.md#supported-package-managers
```

Comment:
This doesn't look right, it should be something like
```suggestion
[supported package managers]: corepack.md#corepack_supported_package_managers
```

Reply:
Hm I think I got lint errors with underscores - will try again with the `corepack` prefix